dawn-1/evm/genesis.json

```diff
@@ -0,0 +1,46 @@
+{
+  "config": {
+      "chainId": 16604737732183,
+      "homesteadBlock": 0,
+      "eip150Block": 0,
+      "eip155Block": 0,
+      "eip158Block": 0,
+      "byzantiumBlock": 0,
+      "constantinopleBlock": 0,
+      "petersburgBlock": 0,
+      "istanbulBlock": 0,
+      "berlinBlock": 0,
+      "londonBlock": 0,
+      "shanghaiTime": 0,
+      "terminalTotalDifficulty": 0,
+      "terminalTotalDifficultyPassed": true,
+      "ethash": {},
+      "astriaOverrideGenesisExtraData": true,
+      "astriaSequencerInitialHeight": 65078,
+      "astriaRollupName": "flame-dawn-1",
+      "astriaCelestiaInitialHeight": 2951695,
+      "astriaCelestiaHeightVariance": 15000,
+      "astriaBridgeAddresses": [
```

Comment:
I'm assuming we'll update in a follow up PR to add the USDC bridge?

Reply:
correct.
Note that this genesis change does not affect block hashes in any way